packages/multi-test/src/test_helpers/contracts/echo.rs

```diff
@@ -48,12 +57,14 @@ fn reply(_deps: DepsMut, _env: Env, msg: Reply) -> Result<Response, StdError> {
     if let Reply {
         result:
             ContractResult::Ok(SubMsgExecutionResponse {
-                data: Some(data), ..
+                data: Some(data),
+                events,
+                ..
             }),
         ..
     } = msg
     {
-        Ok(Response::new().set_data(data))
+        Ok(Response::new().set_data(data).add_events(events))
```

Comment:
I would actually not add these to the `reply` call. These should be appended by the runtime when there is a SubMsg executed without the reply blob doing anything